chanbackup/backupfile.go

```diff
@@ -161,6 +161,9 @@ func (b *MultiFile) ExtractMulti(keyChain keychain.KeyRing) (*Multi, error) {
 	if err != nil {
 		return nil, err
 	}
+	if err := b.mainFile.Close(); err != nil {
```

Comment:
can this be deferred?

Reply:
Any reason to not use `ioutil.ReadFile`?

Reply:
Switched to use `ioutil.ReadFile` confirmed that it uses a defer close internally.